candies/cli/parsers/standard.py

```diff
@@ -137,12 +138,12 @@ def prefixes_of(x: Any) -> Tuple[str, str]:
     def is_(x: Any, of: Type) -> bool:
         return isinstance(x, of) or x == of
 
-    def configure_arg(x: inspect.Parameter, config: Dict):
+    def configure_arg(x: inspect.Parameter, type: Type, config: Dict):
         parser.add_argument(x.name, **config)
 
-    def configure_named_arg(x: inspect.Parameter, config: Dict):
-        short = short_of(x.annotation)
-        short_prefix, long_prefix = prefixes_of(x.annotation)
+    def configure_named_arg(x: inspect.Parameter, type: Type, config: Dict):
```

Comment:
I think `type` should be of type `Any`, because it's passed from `parameter.annotation` attribute, which can be either a type `Arg`, or an instantiated object from type `Arg`.
If `type` is of type `Any`, shouldn't it be `annotation` or something?

Reply:
Yes, you are right! I'll rename `type` to `kind` (as in `Typed`) and change the annotation to `Any`.